### PR TITLE
Add cordova getting started guide

### PR DESCRIPTION
**Pull request by craastad**
Main changes:
* reordered pages based on perceived customer interest
* added cordova getting started guide
* simplified ionic guide while I was there

Screen Shot 2021-04-30 at 13 35 16

### CODE REVIEW

File: content/getting-started/building-a-cordova-app.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+
+Note that you need to set up a [webhook](../building/webhooks) for automatic build triggers. Click the **Create webhook** button on the right sidebar in app settings to add a webhook (not available for apps added via SSH/HTTP/HTTPS).
+
+{{</notebox>}}
+
+## Code signing and publishing Cordova Android and iOS apps
+
+To build, code sign, and publish Cordova Android and iOS apps:
+
+* All Android apps need to be signed before release. See the [Android codesigning docs](../code-signing/android-code-signing/) or the sample workflow below for more details.
```
Comment (helinanever) [inline content/getting-started/building-a-cordova-app.md:30]: codesigning or code signing?

Comment (craastad) [reply content/getting-started/building-a-cordova-app.md:30]: good catch, it seems like it's two words everywhere. I'll make that fix. :+1:

File: content/getting-started/building-an-ionic-app.md [hunk 2]
```diff
@@ -19,20 +19,19 @@ The apps you have available on Codemagic are listed on the Applications page. Se
...
 
 * The code for testing an Ionic Android app also goes under `scripts`. A few examples of testing can be found [here](../testing-yaml/testing).
-* All Android applications need to be signed before release. For Gradle code signing configuration for **Ionic Capacitor** apps refer to the [documentation](../code-signing/android-code-signing/). More information about code signing with YAML in general is [here](../code-signing-yaml/signing). If you are building **Ionic Cordova** Android apps see the workflow sample below.
-* All iOS applications need to be signed before release. For iOS code signing configuration refer to the iOS code signing documentation [here](../code-signing/ios-code-signing/)
-* All generated artifacts can be published to external services. The available integrations currently are email, Slack and Google Play. It is also possible to publish elsewhere with custom scripts (e.g. Firebase App Distribution). Script examples for all of them are available [here](../publishing-yaml/distribution/).
+* All Android apps need to be signed before release. See the [Android codesigning docs](../code-signing/android-code-signing/) or the sample workflow below for more details.
```
Comment (helinanever) [inline content/getting-started/building-an-ionic-app.md:32]: codesigning or code signing=?

**Review by helinanever — APPROVED** [review]
Added a minor spelling-related comment, otherwise looks good to me!

**Pull request merged**